src/main/java/duke/Storage.java

```diff
@@ -0,0 +1,126 @@
+package duke;
+
+import java.io.File;
+import java.io.FileNotFoundException;
+import java.io.FileWriter;
+import java.io.IOException;
+import java.util.Scanner;
+
+/**
+ * Deals with loading tasks from the file and saving tasks in the file.
+ */
+public class Storage {
+    private static String filePath;
+
+    private boolean exit = false;
```

Comment:
I think the boolean variable should be name to something that sounds like booleans? Maybe shouldExit or something?

Reply:
Thanks for pointing this out, will do the changes!